lib/interfaces/include/InverterInterface.h

```diff
@@ -1,17 +1,89 @@
 #ifndef INVERTERINTERFACE_H
 #define INVERTERINTERFACE_H
+#include <stdint.h>
 
-#include "shared_types.h"
+#include "FlexCAN_T4.h"
 
+#include <hytech.h>
+#include "DrivetrainSystem.h"
+#include <CANInterface.h>
+#include <shared_types.h>
 
-// struct InverterMotorControl_s
-// {
+struct InverterParams_s
+{   
+    float MINIMUM_HV_VOLTAGE; 
+};
 
-// }
+/**
+ * Struct containing id info for this specific inverter interface
+ */
+struct InverterIds_s
```

Comment:
nit: I'd prefer these were named CAN_ids instead of just IDs, but feel free to ignore

Reply:
done